pkg/auth/context_test.go

```diff
@@ -126,13 +130,6 @@ func TestContext_GetOrgIdFromClaims(t *testing.T) {
 			},
 			want: "Test Tenant ID",
 		},
-		{
-			name: "Should return empty when orgId != tenantIdClaim",
-			claims: jwt.MapClaims{
-				tenantIdClaim: "Test Tenant ID",
-			},
-			want: "Test Tenant ID",
-		},
 		{
 			name: "Should return when alternateTenantIdClaim",
```

Comment:
Not related to your changes, but the name of this test case doesnt seem complete. Can you fix this as well? thanks

Reply:
Thanks @Rajagopalan-Ranganathan